server/app.js

```diff
@@ -1,8 +1,21 @@
 import express from 'express';
+import morgan from 'morgan';
 import swaggerRouter from './swagger/index';
+import authRoute from './routes/auth';
 
 const app = express();
 
 app.use('/api-docs', swaggerRouter);
+app.use(morgan('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use('/', authRoute);
+
+app.use((req, res) =>
```

Comment:
Let's create 404 file in the middeware folder and put this code there. Let's keep the app.js file clean

Reply:
okay